interfaces/dom.idl

```diff
@@ -44,7 +44,7 @@ dictionary CustomEventInit : EventInit {
 };
 
 
-//[Exposed=(Window,Worker)]
+[Constructor/*, Exposed=(Window,Worker)*/]
```

Comment:
Nope. But, I figured it was best to stay consistent, and if someone wanted to go through and update that file en masse, they could.